Tweetinvi.Core/Public/Models/Enum/Language.cs

```diff
@@ -3,233 +3,198 @@
-        [Language("fi")] Finnish,
-        [Language("fr")] French,
-        [Language("ff")] Fula,
-        [Language("ff")] Fulah,
-        [Language("ff")] Pulaar,
-        [Language("ff")] Pular,
-        [Language("gl")] Galician,
-        [Language("ka")] Georgian,
-        [Language("de")] German,
-        [Language("el")] Greek,
-        [Language("el")] Modern,
-        [Language("gn")] Guaraní,
-        [Language("gu")] Gujarati,
-        [Language("ht")] Haitian,
-        [Language("ht")] HaitianCreole,
-        [Language("ha")] Hausa,
-        [Language("he")] Hebrew,
-        [Language("hz")] Herero,
-        [Language("hi")] Hindi,
-        [Language("ho")] HiriMotu,
-        [Language("hu")] Hungarian,
-        [Language("ia")] Interlingua,
-        [Language("id")] Indonesian,
-        [Language("ie")] Interlingue,
-        [Language("iw")] Ivrit,
-        [Language("ga")] Irish,
-        [Language("ig")] Igbo,
-        [Language("ik")] Inupiaq,
-        [Language("io")] Ido,
-        [Language("is")] Icelandic,
-        [Language("it")] Italian,
-        [Language("iu")] Inuktitut,
-        [Language("ja", "jp")] Japanese,
-        [Language("jv")] Javanese,
-        [Language("kl")] Kalaallisut,
-        [Language("kl")] Greenlandic,
-        [Language("kn")] Kannada,
-        [Language("kr")] Kanuri,
-        [Language("ks")] Kashmiri,
-        [Language("kk")] Kazakh,
-        [Language("km")] Khmer,
-        [Language("ki")] Kikuyu,
-        [Language("ki")] Gikuyu,
-        [Language("rw")] Kinyarwanda,
-        [Language("ky")] Kyrgyz,
-        [Language("kv")] Komi,
-        [Language("kg")] Kongo,
-        [Language("ko")] Korean,
-        [Language("ku")] Kurdish,
-        [Language("kj")] Kwanyama,
-        [Language("kj")] Kuanyama,
-        [Language("la")] Latin,
-        [Language("lb")] Luxembourgish,
-        [Language("lb")] Letzeburgesch,
-        [Language("lg")] Ganda,
-        [Language("li")] Limburgish,
-        [Language("li")] Limburgan,
-        [Language("li")] Limburger,
-        [Language("ln")] Lingala,
-        [Language("lo")] Lao,
-        [Language("lt")] Lithuanian,
-        [Language("lu")] LubaKatanga,
-        [Language("lv")] Latvian,
-        [Language("gv")] Manx,
-        [Language("mk")] Macedonian,
-        [Language("mg")] Malagasy,
-        [Language("ms")] Malay,
-        [Language("ml")] Malayalam,
-        [Language("mt")] Maltese,
-        [Language("mi")] Maori,
-        [Language("mr")] Marathi,
-        [Language("mh")] Marshallese,
-        [Language("mn")] Mongolian,
-        [Language("na")] Nauru,
-        [Language("nv")] Navajo,
-        [Language("nv")] Navaho,
-        [Language("nb")] NorwegianBokmal,
-        [Language("nd")] NorthNdebele,
-        [Language("ne")] Nepali,
-        [Language("ng")] Ndonga,
-        [Language("nn")] NorwegianNynorsk,
-        [Language("no")] Norwegian,
-        [Language("ii")] Nuosu,
-        [Language("in")] Indian,
```

Comment:
I believe "in" is actually a deprecated code for Indonesian, and I don't believe Indian is a language (although I'm certainly no expert!).